deepchem/models/sklearn_models/__init__.py

```diff
@@ -21,29 +22,34 @@
     LassoCV, BayesianRidge
 ]
 
+logger = logging.getLogger(__name__)
+
 
 class SklearnModel(Model):
-  """
-  Abstract base class for different ML models.
+  """Wrapper class that wraps scikit-learn models as DeepChem models.
+
+  When you're working with scikit-learn and DeepChem, at times it can
+  be useful to wrap a scikit-learn model as a DeepChem model. The
+  reason for this might be that you want to do an apples-to-apples
+  comparison of a scikit-learn model to another DeepChem model, or
+  perhaps you want to use the hyperparameter tuning capabilities in
+  `dc.hyper`. The `SklearnModel` class provides a
```

Comment:
What does it provide?

Reply:
Ah forgot to finish this description. Will flesh out this docstring!